### DISCUSSION TITLE
Models

### DISCUSSION THREAD

**Discussion by Shashuface** [Anjok07/ultimatevocalremovergui#1096] [General]
I am working on making karaoke tracks. What are the best
models to use in UVR5, that would remove the main vocals, produce the best
instrumentals and keep the background vocals? I have tried different ensemble
combinations, but I am still hearing a lot of noise.

**Comment by morrorless** [comment]
I've just started using this. the best I've found so far, and I think it's really good is an ensemble of MDX-NET Kim Vocals1, MDX-Net: UVR-MDX Inst HQ 3. MDX-Net: UVR-MDX Inst Main and Demucs v4 htdemucs_ft. Maybe all the loud music over the years but I don't hear any noise or artifacts of what was removed.

**Comment by RogelioServin** [comment]
Here is the model I been using, It has worked well for me, so maybe give it a try.

Note: I updated UVR5 today, Feb 08, 2024 and it still worked great.

<img width="665" alt="Screenshot 2024-02-08 at 12 23 56 PM" src="https://github.com/Anjok07/ultimatevocalremovergui/assets/112532960/8bfe5673-8703-4c0a-a736-bb1b3789158f">

**Comment by leidige** [comment]
For UVR the usual karaoke path is still two-pass: Kim Vocal 2 (or MDX) to get all vocals vs instrumental, then a karaoke model (MDX Karaoke 2 or 5_HP-Karaoke-UVR) on the vocal stem so lead comes out as Vocals and backing as the 'instrumental' of that pass. Mix that backing back with the first instrumental if you want a karaoke bed with harmonies left in. I also ship a Windows local splitter (MO Mirror, https://pc.1momirr.com/ ) that treats backing as part of the accompaniment instead of a third stem - 30-song trial, no upload. It will not beat a dedicated karaoke model on messy stacks; UVR two-pass is still the more controllable route when you already have the models downloaded.
